src/lib/list/list.ts

```diff
@@ -92,7 +92,7 @@ export class MatListSubheaderCssMatStyler {}
 /** An item within a Material Design list. */
 @Component({
   moduleId: module.id,
-  selector: 'mat-list-item, a[mat-list-item]',
+  selector: 'mat-list-item, a[mat-list-item], button[mat-list-item]',
```

Comment:
One last thing that occurred to me just now- we should default all buttons with `mat-list-item` to have `type="button"`. By default, `<button>` elements are `type="submit"`, which you would never really want in an action list.